email_from_template/send_mail.py

```diff
@@ -272,7 +287,11 @@ def render(component, fail_silently=False):
     if not send_mail:
         return mail
 
-    return mail.send()
+    if on_commit:
+        transaction.on_commit(mail.send)
```

Comment:
No, because we're passing `send` which is a bound method with a reference to `mail`.